Proof Luminary099/UPDATE_PROGRAM.agc (pg: 1386-1396) #650

Hexman768 opened the pull request:
### Description
* Proofed pages 1386-1396 and made changes accordingly
* This change closes #650 

<!-- Proofing PRs:
1. Title should follow "Proof [FILE NAME] #[PROOF ISSUE]", e.g:
    "Proof ALARM_AND_ABORT #564"
2. Mention pages checked if you have not proofed the entire file

--><!-- New README/CONTRIBUTING PRs:
1. Title should follow "Add [LANGUAGE] [README|CONTRIBUTING]", e.g:
    "Add Dutch README"
-->

Hexman768 commented:
@wopian @chrislgarry Please review these changes and let me know if they are satisfactory or if additional changes are required, thanks a ton!

Hexman768 commented:
@wopian Just a reminder to review these changes, thanks in advance! 

Also, please let me know if there is a spot available for another repository maintainer, as I would love to take on the work and I have the extra time to put in to contributing also!

wopian commented on Luminary099/UPDATE_PROGRAM.agc:
```diff
@@ -323,28 +330,29 @@ UPSTORE		EQUALS			# GROUND HAS VERIFIED UPDATE.  STORE DATA.
 UPFNDVAC	CAF	CHRPRIO		# (USE EXTENDED VERB PRIORITY)
 		TC	FINDVAC		# GET VAC AREA FOR 'CALL INTSTALL'
 		EBANK=	TEPHEM
 		2CADR	UPJOB		# (NOTE:  THIS WILL ALSO SET EBANK FOR
```
```suggestion
		2CADR	UPJOB		# (NOTE: THIS WILL ALSO SET EBANK FOR
```

wopian replied:
![image](https://github.com/user-attachments/assets/49c1c42e-32de-4108-89d2-841b11e5bd3f)

wopian commented on Luminary099/UPDATE_PROGRAM.agc:
```diff
@@ -299,14 +306,14 @@ UPVRFYNV	VN	2102		# VERB 21 NOUN 02
 UPDTPHAS	EQUALS	FIVE
 
-# PRE-STORE AND FAN TO APPROPRIATE BRANCH SEQUENCE
+# PRE-STORE AND FAN TO APPROPRI ATE BRANCH SEQUENCE
```
```suggestion
# PRE-STORE AND FAN TO APPROPRIATE BRANCH SEQUENCE
```
There is no space here, only a slight misprint on the `A`, `AT` is still occupying 2 character widths

![image](https://github.com/user-attachments/assets/30a6cec8-2165-4a03-a096-5b9b6f7c8f2c)

Hexman768 replied:
Gotcha, applying that change.

wopian commented on Luminary099/UPDATE_PROGRAM.agc:
```diff
@@ -149,17 +155,20 @@
  +1		TS	UPVERBSV	# SAVE UPVERB UNTIL IT'S OK TO ENTER P27
 
-		TC	TESTXACT	# GRAB DISPLAY IF AVAILABLE, OTHERWISE
+		TC	TESTXACT	# GRAB DISPLAY IF AVAILABLE. OTHERWISE
```
```suggestion
		TC	TESTXACT	# GRAB DISPLAY IF AVAILABLE, OTHERWISE
```

This is a comma in the scans 
![image](https://github.com/user-attachments/assets/856b6172-4bb3-43d7-b113-e095c0c4bfc8)

Hexman768 replied:
Ah, I was not able to tell if that was a comment or a period, adding that change also.

wopian commented:
Overall the PR looks good, just a few mistakes that can be corrected 👍

Hexman768 commented:
@wopian I added your suggested changes and added one more of my own that I noticed I missed. 

I am also curious to know how one can become a maintainer on this project, or even just assist in some of the work/reviews?

Hexman768 commented:
@wopian Just a reminder to review the latest changes when you get a minute, thanks!

Hexman768 commented:
@wopian Can this change be merged?

Hexman768 commented:
@wopian Just a reminder to complete this review/merge this PR when you can, thanks!

Hexman768 commented:
@chrislgarry @wopian Is there availability for another maintainer/contributor for this project? I would be more than happy to help with the work load!

Hexman768 commented:
@wopian Can this PR be merged? 

Also, is it possible to contact you, @chrislgarry, or somebody else to see about becoming a project maintainer?

Hexman768 commented:
@wopian Is this change good to be merged?

Hexman768 commented:
@oldmud0 Are you also able to review these changes, it is always good to get more sets of eyes on a code change.

Hexman768 commented:
Update: I just amended the latest commit to match my current GitHub profile.

Hexman768 commented:
@wopian Are these changes good to be merged?

Hexman768 commented:
@wopian Can these changes be merged?

Hexman768 commented:
@wopian @oldmud0 @chrislgarry Can you please review these changes?

Hexman768 commented:
@wopian Am I reaching out to the correct person to get these changes merged? Are we supposed to be able to merge our own pull requests after initial review? 

I read through the contributing guidelines and everything looks correct, I just want to make sure I'm not doing something wrong here.

chrislgarry commented:
Hey Zach, thanks for the PRs. We will get to these when we free up.
Everyone is a volunteer and we have more than full-time obligations
external to this project, and it is a particularly busy time at NASA.
Thankfully we are not under any time pressure, but cant have more than two
contributors so if your goal with your PRs is to be a
contributor/maintainer, I think Virtual AGC could really use your help and
would be happy to have you on board.

On Sun, Jun 8, 2025, 00:06 Zachary Pedigo ***@***.***> wrote:

> *Hexman768* left a comment (chrislgarry/Apollo-11#937)
> <https://github.com/chrislgarry/Apollo-11/pull/937#issuecomment-2953512048>
>
> @wopian <https://github.com/wopian> Am I reaching out to the correct
> person to get these changes merged? Are we supposed to be able to merge our
> own pull requests after initial review?
>
> I read through the contributing guidelines and everything looks correct, I
> just want to make sure I'm not doing something wrong here.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/chrislgarry/Apollo-11/pull/937#issuecomment-2953512048>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAQZKQQMSMERAKI5WET6RO33COZEJAVCNFSM6AAAAAB357426CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDSNJTGUYTEMBUHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
